databases/week1/SQL_class_HomeWORK/sql_homeworkWeek1.sql

```diff
@@ -0,0 +1,70 @@
+USE hyf_lesson1;
+
+-- Find out how many tasks are in the task table
+SELECT COUNT(id)
+FROM task;
+
+-- Find out how many tasks in the task table do not have a valid due date
+SELECT COUNT(id)
+FROM task
+WHERE due_date IS NULL;
+
+-- Find all the tasks that are marked as done
+SELECT task.id, task.title, task.description, status.name
+FROM task
+INNER JOIN status ON status.id=task.status_id
+WHERE status.name="done";
+
+-- Find all the tasks that are not marked as done
+SELECT task.id, task.title, task.description, status.name
+FROM task
+INNER JOIN status ON status.id=task.status_id
+WHERE status.name!="done";
+
+-- Get all the tasks, sorted with the most recently created first
+SELECT *
```

Comment:
Avoid * selection. In some cases select * can impact how rows are locked. It is better to explicitly write the column names you want

Reply:
will make the changes